objectbox/lib/src/native/browser.dart

```diff
@@ -0,0 +1,81 @@
+import 'dart:ffi';
+
+import 'bindings/bindings.dart';
+import 'bindings/helpers.dart';
+import 'store.dart';
+
+/// Data browser allows you to explore the DB contents in a regular web browser.
+///
+/// The Browser runs directly on your device or on your development machine.
+/// Behind the scenes this works by bundling a simple HTTP browser into
+/// ObjectBox when building your app. If triggered, it will then provide a basic
+/// web interface to the data and schema.
+class Browser {
+  late Pointer<OBX_browser> _cBrowser;
+  late final Pointer<OBX_dart_finalizer> _cFinalizer;
+
+  @pragma('vm:prefer-inline')
+  Pointer<OBX_browser> get _ptr => (_cBrowser.address != 0)
+      ? _cBrowser
+      : throw StateError('Browser already closed');
+
+  /// Whether the loaded ObjectBox native library supports ObjectBrowser.
+  static bool isAvailable() => C.has_feature(OBXFeature.ObjectBrowser);
+
+  /// Creates a sync client associated with the given store and options.
+  /// This does not initiate any connection attempts yet: call start() to do so.
```

Comment:
"sync client" and "call start()", forgot to update docs?

Reply:
fixed